### ISSUE TITLE
[Bug]: Load data runtime error

### ISSUE THREAD

**Issue by mklzl** [matrixorigin/matrixone#5274]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): LastCommitId=8491eef04d6b38c7d706f7e789c576e6b63bcbfd
- Hardware parameters: linux
- OS type:centos 7.9
- Others:
```

### Actual Behavior

```
mysql> load data infile '/home/tpch/tpch-dbgen/part.tbl' into table PART FIELDS TERMINATED BY '|' OPTIONALLY ENCLOSED BY '"' LINES TERMINATED BY '\n';
ERROR 20101 (HY000): internal error: panic runtime error: index out of range [1] with length 1: goroutine 1698 [running]:
runtime/debug.Stack()
	/home/go/go/src/runtime/debug/stack.go:24 +0x65
github.com/matrixorigin/matrixone/pkg/common/moerr.ConvertPanicError({0x2137920?, 0xc0006e3a40})
	/home/mo/matrixone/pkg/common/moerr/error.go:410 +0x65
github.com/matrixorigin/matrixone/pkg/vm.Run.func1()
	/home/mo/matrixone/pkg/vm/vm.go:57 +0x31
panic({0x2137920, 0xc0006e3a40})
	/home/go/go/src/runtime/panic.go:838 +0x207
github.com/ma
```

### Expected Behavior

``load data success!``

### Steps to Reproduce

```Markdown
1. create table 
CREATE TABLE PART(
P_PARTKEY     INTEGER NOT NULL,
P_NAME        VARCHAR(55) NOT NULL,
P_MFGR        CHAR(25) NOT NULL,
P_BRAND       CHAR(10) NOT NULL,
P_TYPE        VARCHAR(25) NOT NULL,
P_SIZE        INTEGER NOT NULL,
P_CONTAINER   CHAR(10) NOT NULL,
P_RETAILPRICE DECIMAL(15,2) NOT NULL,
P_COMMENT     VARCHAR(23) NOT NULL,
PRIMARY KEY (P_PARTKEY)
);
2. load data
load data infile '/home/tpch/tpch-dbgen/nation.tbl' into table NATION FIELDS TERMINATED BY '|' OPTIONALLY ENCLOSED BY '"' LINES TERMINATED BY '\n';
```

### Additional information

```
2022/09/27 01:05:34.757314 -0400 ERROR frontend/util.go:461 Connection id: 1001 Status: fail Statement: load data infile /home/tpch/tpch-dbgen/part.tbl into table part fields terminated by | optionally enclosed by " lines terminated by 
2022/09/27 01:05:57.065959 -0400 ERROR frontend/mysql_cmd_executor.go:2376 internal error: panic runtime error: index out of range [1] with length 1: goroutine 1698 [running]:
```

**Comment by jianwan0214** [comment]
can you occur this question repeatedly? And I want to contact with you to discuss this question. @mklzl

**Comment by mklzl** [comment]
of couse you can .  You can contact me through 猫猫

**Comment by aressu1985** [comment]
fix:005e131ca5cf683ec08c21f6c93f29c826fa4178

Loading data/1/part.tbl in to table part,please wait.....
load data infile '/home/sudong/mo-tpch/data/1/part.tbl' into table tpch_1g.part FIELDS TERMINATED BY '|' LINES TERMINATED BY '\n';
mysql: [Warning] Using a password on the command line interface can be insecure.
The data for table part has been loaded successfully,,and cost: 661
